### ISSUE TITLE
Line length not respected in this example

### ISSUE THREAD

**Issue by ry** [dsherret/dprint#66]
dprint_plugin_typescript made the following edit
```diff
-export const BodyUsedError =
-  "Failed to execute 'clone' on 'Body': body is already used";
+export const BodyUsedError = "Failed to execute 'clone' on 'Body': body is already used";
```
this was with this config:
```
    .line_width(80)
    .indent_width(2)
    .next_control_flow_position(dprint::NextControlFlowPosition::SameLine)
    .force_multi_line_parameters(true)
    .force_multi_line_arguments(true)
    .binary_expression_operator_position(dprint::OperatorPosition::SameLine)
```
Note that `export const BodyUsedError = "Failed to execute 'clone' on 'Body': body is already used";` is  88 columns.

**Comment by dsherret** [comment]
@ry thanks for reporting! Was missing support for a possible line break there.

Fixed and published in 0.3.0-alpha.2 (I am using this incrementing alpha version during these iterations)

**Comment by dsherret** [comment]
@ry oops! Missed something. Use `0.3.0-alpha.3` (sorry)
